src/main/java/cn/nukkit/Server.java

```diff
@@ -2420,13 +2420,12 @@ public boolean loadLevel(String name) {
         }
         Map<Integer, LevelConfig.GeneratorConfig> generators = levelConfig.generators();
         for (var entry : generators.entrySet()) {
-            String levelName = name + " Dim" + entry.getKey();
-            if (this.isLevelLoaded(levelName)) {
+            if (this.isLevelLoaded(name)) {
```

Comment:
Reject this change, which is broken the level store because we need the dim id as a key to distinguish between different Levels

Reply:
but the dimension is still there, just more in the name.

Reply:
There is a `Map` storing level in the `Server` class, check them!

Reply:
We have to distinguish the levels in this named way, because `leveldb` correspond to 3 dimensions, which corresponds to 3 `Level` instances.

Reply:
we can't retrieve it from config.json in the created world ?

Reply:
@CoolLoong

Reply:
yes

Reply:
we can't retrieve it from config.json in the created world ?

Reply:
Unless you're going to refactor the system, but I'm not, it's a legacy from NK and requires a lot of change to the method

Reply:
![image](https://github.com/PowerNukkitX/PowerNukkitX/assets/82237910/86dac94e-b00f-4a65-b7d3-4266b8531108)

Reply:
what pushed me to change this is the teleportation command from world to world which needs Dim0 to be teleported.
I'll try another way

Reply:
It's easy to use it conveniently, just make a soft enum as a complement, I'll finish it afterward